skrub/datasets/tests/test_fetching.py

```diff
@@ -219,3 +220,39 @@ def test_fetch_movielens():
             )
             mock_urlretrieve.assert_not_called()
             assert disk_loaded_info == returned_info
+
+
+def test_fetch_credit_fraud():
```

Comment:
Actually, I'm wrong.

Here, you are testing something that request `pyarrow` but this is an optional one. So you should add:

```
pytest.importorskip("pyarrow")
```

to skip the test for when we don't have `pyarrow` installed.

Reply:
Thanks @glemaitre !